{{cookiecutter.project_slug}}/backend/app/api/api_v1/routers/auth.py

```diff
@@ -34,3 +34,30 @@ async def login(
     )
 
     return {"access_token": access_token, "token_type": "bearer"}
+
+
+@r.post("/signup")
+async def signup(
+    db=Depends(get_db), form_data: OAuth2PasswordRequestForm = Depends()
+):
+    user = sign_up_new_user(db, form_data.username, form_data.password)
+    if not user:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
```

Comment:
do you think 403 is the right status code for things that already exist? is that what FastAPI recommends?

Reply:
I often see 422 as a catch all for validation errors, 409 is fine too you just don't see it as often.

Reply:
I'll change it so, along with the monkeypatch :+1:

Reply:
HTTP_422_UNPROCESSABLE_ENTITY

Reply:
Okay, that's fine.